### DISCUSSION TITLE
Check if downloading

### DISCUSSION THREAD

**Discussion by ghost** [RandomNinjaAtk/docker-lidarr-extended#165] [Q&A] [answered]
I installed Lidarr extended as a container on my Raspberry Pi. I got it running and it just downloaded a newly released track by an artist in my music folder. I would like to know if there is a way to see if Lidarr is downloading other files from other artists? I added another artist to Lidarr and it doesnt seem to download.

Maybe installing this on a raspberry was a bad idea, or I don't know enough about Lidarr. But it just worked with a Single and I want it to work with other tracks aswell :)

Does someone here have infos that would be usefull for a noob like me? ^^

**Comment by RandomNinjaAtk** [comment, marked as answer]
You need to review the logs. Watch the docker container console log to get a live view. Or in Lidarr Web (System -> Log Files) and open the relevant log file.

**Reply by ghost** [reply]
Thank you, will do :)
